File: Alodjinha/app/src/main/java/br/com/lodjinha/alodjinha/model/Produto.java
package br.com.lodjinha.alodjinha.model;

import android.graphics.Bitmap;
import android.text.Spanned;

import java.io.Serializable;
import java.util.List;

/**
 * Created by douglasromano on 15/02/2017.
 */

public class Produto implements Serializable {

    private int id;

    private String nome;

    private String urlImagem;

    private Spanned descricao;

    private double precoDe;

    private double precoPor;

    private Bitmap imgProduto;

    private List<Categoria> categorias;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getUrlImagem() {
        return urlImagem;
    }

    public void setUrlImagem(String urlImagem) {
        this.urlImagem = urlImagem;
    }

    public Spanned getDescricao() {
        return descricao;
    }

    public void setDescricao(Spanned descricao) {
        this.descricao = descricao;
    }

    public double getPrecoDe() {
        return precoDe;
    }

    public void setPrecoDe(double precoDe) {
        this.precoDe = precoDe;
    }

    public double getPrecoPor() {
        return precoPor;
    }

    public void setPrecoPor(double precoPor) {
        this.precoPor = precoPor;
    }

    public List<Categoria> getCategorias() {
        return categorias;
    }

    public void setCategorias(List<Categoria> categorias) {
        this.categorias = categorias;
    }

    public Bitmap getImgProduto() {
        return imgProduto;
    }

    public void setImgProduto(Bitmap imgProduto) {
        this.imgProduto = imgProduto;
    }
}

File: app/src/main/java/br/com/b2w/desafio/model/produto/Produto.kt
package br.com.b2w.desafio.model.produto

import android.os.Parcelable
import br.com.b2w.desafio.model.Categoria
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Produto (
    @SerializedName("id") val id: Int? = null,
    @SerializedName("nome") val nome: String? = null,
    @SerializedName("urlImagem") val urlImagem: String? = null,
    @SerializedName("descricao") val descricao: String? = null,
    @SerializedName("precoDe") val precoDe: Double? = null,
    @SerializedName("precoPor") val precoPor: Double? = null,
    @SerializedName("categoria")val categoria: Categoria? = null
) : Parcelable